manifests/params.pp

```diff
@@ -116,6 +116,6 @@
   $puppi_helper = 'standard'
   $debug = false
   $audit_only = false
-  $noops = undef
+  $noops = false
```

Comment:
Neither `true` nor `false` is applicable here. `true` doesn't make the changes at all, `false` makes them applied even with `--noop`. The whole noops params must be removed. I did that already and will push the changes.